components/NavigationBar/tests/unit/NavigationBarAccessibilityMutatorTestColorTest.swift

```diff
@@ -15,8 +15,13 @@
  */
 
 import XCTest
-import MaterialComponents
+import MaterialComponents.MaterialNavigationBar
+
+#if IS_BAZEL_BUILD
+import material_text_accessibility_ios_MDFTextAccessibility
+#else 
 import MDFTextAccessibility
+#endif // IS_BAZEL_BUILD
```

Comment:
We should be able to drop this and the flag with 4cb991b649ae856de73cb217b518bd50f539016a landed.